features/fixtures/ios-swift-cocoapods/iOSTestApp/scenarios/OOMDefaultscenario.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Created by Jamie Lynch on 06/03/2018.
+// Copyright (c) 2018 Bugsnag. All rights reserved.
+//
+
+import Foundation
+import Bugsnag
+
+/**
+ * Sends a handled Error to Bugsnag
+ */
+class OOMDefaultScenario: OOMBaseScenario {
```

Comment:
Why can't I see where OOMBaseScenario is defined?  Should the other classes inherit from it?

Reply:
Sorry, that file was removed from the project but not the repo

Reply:
Can you spell the path out to me, I've grepped my checkout and can't see the implementation.

Reply:
Ah, I think I see what you mean now - `OOMDefaultscenario.swift`should have been removed and you have done so now.